### PR TITLE
Adding US_Dvorak support

### PR DESCRIPTION
**Pull request by antijingoist**
Added a copy of the dvorak keyboard layout I'm using in my Cosmos project.

### CODE REVIEW

File: source/Cosmos.System2/Keyboard/ScanMaps/USDvorakLayout.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            _keys.Add(new KeyMapping(0x0B, '0', ')', '0', '0', ')', '0', ConsoleKeyEx.D0));
+            /* -, =, Bksp, Tab */
+            _keys.Add(new KeyMapping(0x0C, '[', '{', '[', '{', '[', '{', ConsoleKeyEx.LBracket));
+            _keys.Add(new KeyMapping(0x0D, ']', '}', ']', '}', ']', '}', ConsoleKeyEx.RBracket));
+            //_keys.Add(new KeyMapping(0x0C, '-', '_', '-', '-', '_', '-', ConsoleKeyEx.Minus));
+            //_keys.Add(new KeyMapping(0x0D, '=', '+', '=', '=', '+', '=', ConsoleKeyEx.Equal));
+            _keys.Add(new KeyMapping(0x0E, ConsoleKeyEx.Backspace));
+            _keys.Add(new KeyMapping(0x0F, ConsoleKeyEx.Tab));
+            /*      QWERTYUIOP[] */
+            _keys.Add(new KeyMapping(0x10, '\'', '"', '\'', '\'', '"', '"', ConsoleKeyEx.Apostrophe)); //'q', 'Q', 'q', 'Q', 'q', 'Q', ConsoleKeyEx.Q));
```
Comment (quajak) [inline source/Cosmos.System2/Keyboard/ScanMaps/USDvorakLayout.cs:52]: Could you please remove the commented out code in the following lines?

**Review by quajak — CHANGES_REQUESTED** [review]
Thank you for the PR!

It is currently not compiling since the variable was renamed from `_keys` to `Keys`.

File: source/Cosmos.System2/Keyboard/ScanMaps/USDvorakLayout.cs (hunk 1 above)
Comment (antijingoist) [reply source/Cosmos.System2/Keyboard/ScanMaps/USDvorakLayout.cs]: done :)

**Comment by antijingoist** [comment]
Updated the variable names.

**Review by quajak — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**